share/spice/tlds/TLDs.js

```diff
@@ -0,0 +1,43 @@
+(function(env) {
```

Comment:
@javathunderman the filename needs to be lowercased here -- all JS and CSS filenames are always lowerced.